JavaScript/Next-js.md

```diff
@@ -0,0 +1,231 @@
+## Introduction
+
+Next.js supports some very cool features which can be used on different kind of projects. For example, some projects with public pages couldn't use create-react-app because it doesn't support server-side rendering, which makes SEO much harder. So in projects that require SSR, the different stack must be used that would probably include something like express on top of node, and custom webpack and babel configuration.
```

Comment:
```suggestion
Next.js supports some very cool features which can be used in different kinds of projects. For example, some projects with public pages can't be built with create-react-app (CRA) because it doesn't support server-side rendering, which makes SEO much harder. So, in projects that require SSR, a different stack must be used that would probably include something like Express on top of node, and custom Webpack and Babel configurations.
```